docs/trace/getting-started-jaeger/getting-started-jaeger.csproj

```diff
@@ -4,7 +4,8 @@
     <ProjectReference Include="$(RepoRoot)\src\OpenTelemetry.Exporter.OpenTelemetryProtocol\OpenTelemetry.Exporter.OpenTelemetryProtocol.csproj" />
     <ProjectReference Include="$(RepoRoot)\src\OpenTelemetry.Instrumentation.Http\OpenTelemetry.Instrumentation.Http.csproj" />
   </ItemGroup>
+
   <ItemGroup>
-    <PackageReference Include="System.Net.Http" Condition="$(TargetFramework) == 'net472' or $(TargetFramework) == 'net48'"/>
+    <Reference Include="System.Net.Http" Condition="$(TargetFramework.StartsWith('net4'))" />
```

Comment:
If you have a sec, I'm curious if `Condition="'$(TargetFrameworkIdentifier)' != '.NETCoreApp'"` would work here.

I see the runtime packages [doing that](https://github.com/dotnet/runtime/blob/bdceab9718962585206bdb309557d986e12a3d55/src/libraries/System.Diagnostics.DiagnosticSource/src/System.Diagnostics.DiagnosticSource.csproj#L99C25-L99C72) but I'm not sure if it is a standard thing or something added by custom targets in their repo.

Reply:
Let me try, thanks for the tips!

Reply:
Updated.